### DISCUSSION TITLE
File Rename not working with Obsidian 1.0

### DISCUSSION THREAD

**Discussion by executive-of-dysfxn** [SilentVoid13/Templater#900] [Help] [answered]
I have a template that used to let me rename the file and move it to a new folder. When I updated to Obsidian 1.0 (currently v1.0.3) I noticed that the file will move to the correct folder but the file will remain as "Untitled." This is true whether I'm creating a new file from the template or open a new file and insert the template. I'm wondering if anyone else is having this issue and how they fixed it. Thanks!

My current template text:

```
await tp.file.rename("Enter File Name")
await tp.file.move("/folder-name" + "/" + tp.file.title)
```

**Comment by AB1908** [comment, marked as answer]
Try ignoring the rename since move can rename as well.

**Reply by executive-of-dysfxn** [reply]
That worked perfectly, thanks!
